yarn-project/acir-simulator/src/avm/index.test.ts

```diff
@@ -0,0 +1,62 @@
+import { Fr } from '@aztec/foundation/fields';
+
+import { mock } from 'jest-mock-extended';
+
+import { AvmMachineState } from './avm_machine_state.js';
+import { AvmStateManager } from './avm_state_manager.js';
+import { AvmInterpreter } from './interpreter/interpreter.js';
+import { Add } from './opcodes/arithmetic.js';
+import { Return } from './opcodes/control_flow.js';
+import { interpretBytecode } from './opcodes/from_bytecode.js';
+import { CalldataCopy } from './opcodes/memory.js';
+import { Opcode } from './opcodes/opcodes.js';
+
+function genInstructionBytecode(opcode: Opcode, args: number[], assertArgsLength: number): Buffer {
```

Comment:
As we have a `from_bytecode` file with its own test, this could probably mirror that layout.